src/Dhall/Import.hs

```diff
@@ -589,7 +598,7 @@ exprFromPath m (Path {..}) = case pathType of
                 exists <- Filesystem.isFile path
                 if exists
                     then return ()
-                    else Control.Exception.throwIO MissingFile
+                    else Control.Exception.throwIO (MissingFile path)
```

Comment:
Is `path` already canonicalized here?

Reply:
Yes.  The part where `exprFromPath` gets called (very indirectly) is here: https://github.com/Profpatsch/dhall-haskell/blob/8b741bae1fba6c7a007949db7f5e8058b5adabc7/src/Dhall/Import.hs#L752